database/neo4j/index.js

```diff
@@ -0,0 +1,224 @@
+const { driver } = require('./db');
+
+
+
+module.exports.getRestaurantReviews = ({ restaurantId }) => {
+  const session = driver.session();
+  // let restString = restaurantId.toString();
+  // console.log(restaurantId);
+  return session
+    .run(`MATCH (rest:Restaurant)<-[r]-(rev:Review)<-[r2]-(reviewer:Reviewer)
+          WHERE rest.restaurantId = $restaurantId
+          MATCH (reviewer)-->(revCount:Review)
+          RETURN
+            count(revCount) AS numReviews,
+            rest.restaurantId AS restaurantId,
+            rest.city AS city,
+            r.tag AS filterTag,
+            rev.reviewId AS reviewId,
+            rev.ambience AS ambience,
+            rev.comments AS comments,
+            rev.food AS food,
+            rev.noise AS noise,
+            rev.overall AS overall,
+            rev.recommend AS recommend,
+            rev.service AS service,
+            r2.date AS dineDate,
+            reviewer.reviewerId AS reviewerId,
+            reviewer.color AS color,
+            reviewer.firstName AS firstName,
+            reviewer.lastName AS lastName,
+            reviewer.vip AS vip
```

Comment:
looks good. i mean it's unfortunate you couldn't pass the order by variable, but still. :)